src/index.js

```diff
@@ -1,4 +1,6 @@
-require('es6-promise').polyfill();  // Polyfill `Promise`.
+// Polyfill `Promise`.
+global.Promise = global.Promise || require('promise-polyfill');
```

Comment:
We should use window instead of global for consistency as we do below in this file.